### PR TITLE
Tips & Tricks refresh

### PR DESCRIPTION
**Pull request by vexr**
- Completely refreshed the Tips & Tricks section.  
- Merged Troubleshooting content into Tips & Tricks, incorporating the single backtrace instructions and added redirects.  
- Created a new button theme component for this section and for implementing cleaner button links in the documentation moving forward.

### CODE REVIEW

**Comment by vexr** [comment]
I have two different styles for the buttons under `Telemetry & Explorers`: a primary style and a secondary variant. Currently, I'm using the primary style for in-house tools and the secondary (white) style for third-party explorers. Do you like this approach, or do you prefer using all primary buttons or all secondary (white) buttons?

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 1]
```diff
@@ -11,243 +11,438 @@ keywords:
...
+--dsn-pending-out-connections <value>
+```
 
-### Record Chunks Reading Mode
+</details>
 
-Upon startup, each farm will [benchmark](#benchmarking-your-farmer) the performance of every plot to identify the most efficient proving method, yielding either `ConcurrentChunks` or `WholeSector` results. If you already know the optimal method for your setup, you can specify it as an argument for each farm to save time benchmarking. 
-For example, you can include record-chunks-mode= after defining the path and size, assigning the desired value, e.g., `path=/mnt/subspace1,size=100G,record-chunks-mode=ConcurrentChunks`. If you do not provide this parameter, the system will benchmark each disk on startup to identify the most efficient method.
+<details>
+<summary><small>Snap Sync</small></summary>
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:109]: I think this description is for the full sync, and what this section should really be focusing on, is the snap sync. 
Randy has updated the description for the snap sync in this PR https://github.com/autonomys/subspace-docs/pull/692, do you mind adding it here? Feel free to leave everything that you've put about the full sync, it reads well

**Review by EmilFattakhov — COMMENTED** [review]
Reads great @vexr, I think the content is great and I just had one small piece of feedback. 
I will glance over it once again before merging it, as it might cause some conflicts with [PR 692](https://github.com/autonomys/subspace-docs/pull/692), but shouldn't be too many

**Comment by EmilFattakhov** [comment]
@vexr I was just about to resolve the conflicts, perfect timing :)

**Review by EmilFattakhov — APPROVED** [review]
LGTM, thank you very much, the readability on that page improved significantly 🚀

**Pull request merged**